Polygon_mesh_processing/examples/Polygon_mesh_processing/CMakeLists.txt

```diff
@@ -84,6 +84,7 @@ create_single_source_cgal_program( "compute_normals_example_Polyhedron.cpp"  CXX
 create_single_source_cgal_program( "compute_normals_example.cpp" CXX_FEATURES cxx_range_for cxx_auto_type )
 create_single_source_cgal_program( "point_inside_example.cpp")
 create_single_source_cgal_program( "triangulate_faces_example.cpp")
+create_single_source_cgal_program( "triangulate_faces_split_visitor_example.cpp")
```

Comment:
I think you forgot to commit the file `triangulate_faces_split_visitor_example.cpp`.